src/models/room-summary.ts

```diff
@@ -14,11 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import type { Hero } from "./room.ts";
+
 export interface IRoomSummary {
     "m.heroes": string[];
     "m.joined_member_count"?: number;
     "m.invited_member_count"?: number;
 }
+export interface IRoomSummaryMSC4186 {
```

Comment:
As usual, this needs documentation; particularly so in this case given it is part of js-sdk's public API